packages/ses-types-test/validation.ts

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable no-unused-vars, no-underscore-dangle */
+/// <reference types="ses"/>
+
+// eslint-disable-next-line prettier/prettier
```

Comment:
Prettier is confused for the next line. It thinks there should be an `=` in there.

Reply:
So I looked into this and it seems that the parser used by the prettier eslint plugin somehow isn't up to date. I have no idea why because the eslint parser itself is fine (`"@typescript-eslint/parser": "^4.25.0"`). In general I'm not a big fan of prettier integrated inside of eslint, and this might be another reason we want to kill it, but that's a battle for another day.